thereadingmachine/scraper/news_scraper/spiders/spiders.py

```diff
@@ -59,8 +105,7 @@ def parse_item(self, response):
         try:
             item['title'] = title
             item['article'] = article
-            item['link'] = response.url.replace(
-                'http://', '').replace('https://', '')
+            item['link'] = response.url
```

Comment:
the cleanup was in place to avoid duplication. I'd suggest to wrap it into a method and invoke it unless you want multiple copies of the same article